client/src/app/contacts/concert/concert-contact.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { ContactService, ToasterService } from '../../_services';
+import { Router } from '@angular/router';
+
+@Component({
+    templateUrl: './concert-contact.component.html',
+})
+
+export class ConcertContactComponent {
+
+    contactForm: FormGroup;
+    token: String;
+
+    constructor(private fb: FormBuilder,
+                private contactService: ContactService,
+                private toasterService: ToasterService,
+                private router: Router) {
+        this.createForm();
+    }
+
+    createForm(): void {
+        this.contactForm = this.fb.group({
+            lastName: new FormControl('', [Validators.required]),
+            firstName: new FormControl('', [Validators.required]),
+            email: new FormControl('', [Validators.required, Validators.email]),
+            phone: new FormControl('', [Validators.required, Validators.pattern(/^((\+)33|0)[1-9](\d{2}){4}$/)]),
```

Comment:
Only french phones ?

Reply:
Nope you are right :) Could you find a valid regex for international phone ?

Reply:
See https://stackoverflow.com/questions/2113908/what-regular-expression-will-match-valid-international-phone-numbers?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa